### PR TITLE
Add modulo tests

### PR DESCRIPTION
**Pull request by alexisaron**

### CODE REVIEW

File: tests/WorkflowTest.php [hunk 1]
```diff
@@ -55,13 +55,13 @@ public function testResultShouldThrowAnExceptionOnUnknownBehavior()
     public function testBaseJsonGeneration()
     {
         $workflow = new \Ezweb\Workflow\Workflow('customWorkflow');
-        $this->assertEquals('{"name":"customWorkflow","value":[]}', json_encode($workflow));
+        $this->assertSame('{"name":"customWorkflow","value":[]}', json_encode($workflow));
```
Comment (azecca) [inline tests/WorkflowTest.php]: For compare json data, I think is good use $this->assertJsonStringEqualsJsonString()
https://phpunit.readthedocs.io/fr/latest/assertions.html#assertjsonstringequalsjsonstring
What do you think?

File: tests/WorkflowTest.php [hunk 2]
```diff
@@ -55,13 +55,13 @@ public function testResultShouldThrowAnExceptionOnUnknownBehavior()
...
     }
 
     public function testBaseJsonGenerationThroughTojsonMethod()
     {
         $workflow = new \Ezweb\Workflow\Workflow('customWorkflow');
-        $this->assertEquals('{"name":"customWorkflow","value":[]}', $workflow->toJson());
+        $this->assertSame('{"name":"customWorkflow","value":[]}', $workflow->toJson());
```
Comment (azecca) [inline tests/WorkflowTest.php]: For compare json data, I think is good use $this->assertJsonStringEqualsJsonString()
https://phpunit.readthedocs.io/fr/latest/assertions.html#assertjsonstringequalsjsonstring
What do you think?

File: tests/WorkflowTest.php [hunk 3]
```diff
@@ -82,15 +82,15 @@ public function testGenerateStringFromSimpleWorkflow()
...
     }
 
     public function testWorkflowCreatedFromJsonOutputTheSameJson()
     {
         $json = file_get_contents(__DIR__ . '/samples/json/completeWorkflow.json');
         $trimmedJson = json_encode(json_decode($json));
         $workflow = \Ezweb\Workflow\Parser::createFromJson($trimmedJson);
-        $this->assertEquals($trimmedJson, json_encode($workflow));
+        $this->assertSame($trimmedJson, json_encode($workflow));
     }
```
Comment (azecca) [inline tests/WorkflowTest.php:94]: For compare json data, I think is good use $this->assertJsonStringEqualsJsonString()
https://phpunit.readthedocs.io/fr/latest/assertions.html#assertjsonstringequalsjsonstring
What do you think?

File: tests/WorkflowTest.php [hunk 4]
```diff
@@ -116,6 +116,6 @@ public function testBuildAWorkflowManually()
         $operator->attachNewVars('channelId');
         $json = file_get_contents(__DIR__ . '/samples/json/manualWorkflowBuild.json');
         $trimmedJson = json_encode(json_decode($json));
-        $this->assertEquals($trimmedJson, json_encode($workflow));
+        $this->assertSame($trimmedJson, json_encode($workflow));
```
Comment (azecca) [inline tests/WorkflowTest.php]: For compare json data, I think is good use $this->assertJsonStringEqualsJsonString()
https://phpunit.readthedocs.io/fr/latest/assertions.html#assertjsonstringequalsjsonstring
What do you think?

File: tests/WorkflowTest.php (hunk 1 above)
Comment (alexisaron) [reply tests/WorkflowTest.php]: Sure !

File: tests/WorkflowTest.php (hunk 2 above)
Comment (alexisaron) [reply tests/WorkflowTest.php]: Sure!

File: tests/WorkflowTest.php (hunk 3 above)
Comment (alexisaron) [reply tests/WorkflowTest.php:94]: Done !

File: tests/WorkflowTest.php (hunk 4 above)
Comment (alexisaron) [reply tests/WorkflowTest.php]: Changed!

**Pull request merged**